mne/coreg.py

```diff
@@ -789,6 +823,22 @@ def _write_mri_config(fname, subject_from, subject_to, scale):
 
 
 def _scale_params(subject_to, subject_from, scale, subjects_dir):
+    """Assemble parameters for scaling
```

Comment:
We usually don't have full stops after the summary line, is this a new convention?